### PR TITLE
Redesigned 'modules' project template to provide both CLI and Web environments.

### PR DESCRIPTION
**Pull request by dschissler**

### CODE REVIEW

**Comment by dschissler** [comment]
I dramatically changed the `modules` project to incorporate a CLI and Web environment and to offer better sharing of services  across the project.

**Comment by dschissler** [comment]
I'll add that I really dislike the existing `modules` template and I think that this new one could carry a developer fairly far along to the point that they could base a long term project off of it.

**Comment by sergeyklay** [comment]
Thanks

File: scripts/Phalcon/Builder/Project/Modules.php [hunk 1]
```diff
@@ -119,18 +143,18 @@ private function createHtaccessFiles()
...
-        $putFile = $this->options->get('projectPath') . 'apps/frontend/views/index/index.phtml';
+        $putFile = $this->options->get('projectPath') . 'app/modules/frontend/views/index/index.phtml';
         $this->generateFile($getFile, $putFile);
 
         return $this;
     }
 
     /**
-     * Creates the configuration
+     * Creates the app/modulesuration
```
Comment (dschissler) [inline scripts/Phalcon/Builder/Project/Modules.php]: TODO:

Comment (sergeyklay) [inline scripts/Phalcon/Builder/Project/Modules.php]: ?

Comment (dschissler) [inline scripts/Phalcon/Builder/Project/Modules.php]: Just a simple typo and I left myself a note to fix it.

**Pull request merged**